extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -74,14 +74,15 @@ export class AmpAd extends AMP.BaseElement {
       const slotId = this.win.ampAdSlotIdCounter++;
       this.element.setAttribute('data-amp-slot-index', slotId);
 
+      const useRemoteHtml = (
+          !(adConfig[type] || {}).remoteHTMLDisabled &&
+            this.win.document.querySelector('meta[name=amp-3p-iframe-src]') &&
+            !this.element.getAttribute('rtc-config'));
```

Comment:
You should move rtc-config check to doubleclick a4a config since it only applies there

Reply:
done